Content.Server/Construction/Conditions/DoorWelded.cs

```diff
@@ -23,15 +23,19 @@ public void ExposeData(ObjectSerializer serializer)
         public async Task<bool> Condition(IEntity entity)
         {
             if (!entity.TryGetComponent(out ServerDoorComponent doorComponent)) return false;
-            return doorComponent.IsWeldedShut;
+            return doorComponent.IsWeldedShut == Welded;
         }
 
         public void DoExamine(IEntity entity, FormattedMessage message, bool inDetailsRange)
         {
             if (!entity.TryGetComponent(out ServerDoorComponent doorComponent)) return;
 
-            if (doorComponent.State == ServerDoorComponent.DoorState.Closed)
+            if (doorComponent.State == ServerDoorComponent.DoorState.Closed && Welded)
                 message.AddMarkup("First, weld the door.\n");
+            else if (doorComponent.IsWeldedShut && !Welded)
+            {
+                message.AddMarkup("First, Weld the door.\n");
+            }
```

Comment:
Mmm, this is wrong.
If the door is welded, and we need the door to be unwelded it should say "First, unweld the door."
If the door is unwelded, and we need the door to be welded it should say "First, weld the door."